ux-guidelines.md

```diff
@@ -0,0 +1,177 @@
+country-specific payment methods (e.g., UPI, PIX, GiroPay, etc). SPC has no
+requirement or limitation on the type of payment instrument being used or
+displayed.
+
+The `displayName` and `details` fields are used to convey textual information
+about the payment instrument. As the names suggest, `displayName` should be set
+to the primary name for the payment instrument, while `details` can be used to
+add additional clarifying details to help the user differentiate or make
+decisions.  For example, for a credit card payment instrument, `displayName`
+may be set to the product name for the card (e.g., "FancyBank Infinity Card"),
+whilst details might include the last four digits of the card and/or expiry
+date (e.g., "1234 01/32").
+
+Note: An implementor may truncate the `displayName` and `details` fields in
+order to fit the text into the available UX space. See issue #269 for
+discussions on setting normative length limits on fields.
+
+The `icon` field is set to the URL of an image representing the payment
+instrument. Implementors are expected to support common image formats such as
+PNG or JPG. The payment instrument icon image is intended to be a quick visual
+confirmation for the user, and is expected to be somewhat small on the screen.
+
+Implementation specifics:
+
+- Google Chrome's (and other Chromium-based browsers) implementation on both
+  desktop and mobile platforms currently linearly scales the image (preserving
+  the input aspect ratio) to fit in a 32dp x 20dp region, i.e., the display
+  region has an aspect ratio of 1.6:1.
+    - Given current average screen densities, we would recommend that
+      integrators pass in a logo with physical pixel size at least 3x of that,
+      i.e., at least 96px by 60px.
+    - The Chrome team have stated that they are interested in hearing from
+      partners if a 32dp x 20dp region is too small, as it may not allow for
+      detailed icons (such as full card art for credit cards).
+
+### Payment entity logos
+
+Relevant fields: [paymentEntitiesLogos], [paymentEntitiesLogos\[x\].url], [paymentEntitiesLogos\[x\].label]
+
+The payment entity logos are intended to communicate to the user which payment
+companies or organizations are involved in the current transaction. This is
+especially helpful if the identities of these parties would not otherwise be
+obvious to the user. Examples include the issuing bank and/or a card network in
+the case of credit/debit card payments, the user's bank and a payment
+initiation services provider (PISP) in Open Banking, etc.
+
+The payment entity logos are expected to be displayed prominently within the
+SPC transaction confirmation UX, but should not significantly overshadow the
+other information shown. Implementors are able to support any number of logos
+passed within `paymentEntityLogos`, and may ignore extra logos beyond what
+they support displaying, but it is recommended that they support up to two
+logos being shown.
```

Comment:
question: does this mean the UserAgent actually does not have to implement this part of the spec at all, and could just drop all entity data? Although we do not need to dictate how they do it, I am opinionated that this privacy context is important and we should demand a minimum bar be met.

This is also conflicting with the information presented in the current spec: `the User Agent MUST ensure that the following is communicated to the user` https://w3c.github.io/secure-payment-confirmation/#sctn-information-presented